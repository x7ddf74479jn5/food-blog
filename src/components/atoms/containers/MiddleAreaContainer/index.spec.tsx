import renderer from "react-test-renderer";

import { MiddleAreaContainer } from ".";

describe("components/atoms/containers/MiddleAreaContainer", () => {
  it("snapshot", () => {
    const tree = renderer.create(<MiddleAreaContainer>children</MiddleAreaContainer>).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
