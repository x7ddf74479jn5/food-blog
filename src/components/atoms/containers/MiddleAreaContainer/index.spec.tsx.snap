// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/atoms/containers/MiddleAreaContainer snapshot 1`] = `
<div
  className="flex flex-col md:flex-row gap-4 md:gap-16 justify-center items-center md:items-stretch mb-16"
>
  children
</div>
`;
